crates/re_viewer/src/ui/view_spatial/ui.rs

```diff
@@ -399,17 +403,16 @@ impl ViewSpatialState {
         highlights: &SpaceViewHighlights,
     ) {
         self.scene_bbox = scene.primitives.bounding_box();
-        // If this is the first time the bounding box is set, (re-)determine the nav_mode.
-        // TODO(andreas): Keep track of user edits
-        if self.scene_bbox_accum.is_nothing() {
+
+        if self.nav_mode.is_auto() {
             self.scene_bbox_accum = self.scene_bbox;
```

Comment:
it's not clear to me why we accumulate the bounding box only if the user have explicitly set a navigation mode. It makes `scene_bbox_accum` a very misleading name

Reply:
silly mistake. fixing